### PR TITLE
[ARTS-255] baseapp move

### PR DESCRIPTION
**Pull request by itye-msft**
moved files to a new sample-service folder.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,11 +9,11 @@
     <relativePath/>  
     <!-- lookup parent from repository --> 
   </parent>  
-  <groupId>com.microsoft</groupId>  
+  <groupId>ndph</groupId>
```
Comment (tamirkamara) [inline pom.xml:12]: Is this the right group? I see on other services: uk.ac.ox.ctsu

**Review by tamirkamara — CHANGES_REQUESTED** [review]
LICENSE and README.md should be on the root folder. perhaps the service should also include a readme specifically for it.

File: pom.xml [hunk 2]
```diff
@@ -9,11 +9,11 @@
     <relativePath/>  
     <!-- lookup parent from repository --> 
   </parent>  
-  <groupId>com.microsoft</groupId>  
+  <groupId>uk.ac.ox.ctsu</groupId>
   <artifactId>java-base-app</artifactId>  
   <version>0.0.1-SNAPSHOT</version>  
   <name>java-base-app</name>  
-  <description>Base project for Azure Spring Boot</description>  
+  <description>Base project for Spring Boot</description>
```
Comment (balteravishay) [inline pom.xml:16]: Spring Boot and Cloud, no? just not **azure** spring cloud.

**Review by balteravishay — APPROVED** [review]
approve with very minor comment

File: pom.xml (hunk 2 above)
Comment (itye-msft) [reply pom.xml]: Done

**Review by tamirkamara — APPROVED** [review]

**Pull request merged**